### ISSUE TITLE
Adding Java installation instructions to OpenRefine Setup page

### ISSUE THREAD

**Issue by dipietroc** [datacarpentry/OpenRefine-ecology-lesson#247]
For Windows PCs, Java is required in order to run OpenRefine. 

The instructions in the OpenRefine setup state that the application is Java based, but doesn't explicitly state that Java has to be installed on the users PC. Additionally, Java has more recently moved to a paid license model. If an organization did not subscribe to the paid Java license, and, depending on whether or not a user is utilizing a workplace computer to run OpenRefine, this may prevent a user from downloading and running the Oracle version of Java on their PC.

I propose adding links to both the Oracle Java download, as well as the OpenRefine GitHub page link below to the setup instructions. The link guides users on how to run OpenRefine with an OpenSource version of Java, called OpenJDK.

[https://github.com/OpenRefine/OpenRefine/wiki/Setup-JAVA](url)

Suggestion for updated text:

OpenRefine is a Java application, and you need to have a ‘Java Runtime Environment’ (JRE) installed on your computer to run OpenRefine. If you don’t already have one installed then you can download and install from [http://java.com](url) by going to the site and clicking “Free Java Download”.

If you are unable to download and run the Oracle version of Java at your institution, you may alternatively use OpenJDK instead, by following the instructions contained in the following link:

[https://github.com/OpenRefine/OpenRefine/wiki/Setup-JAVA](url)

**Comment by CatChapman** [comment]
I agree with this; I was surprised that this wasn't already in the lesson plan if I'm honest.

Checking the webpage for java.com and as of today, there's no button that says "Free Java Download" - it just says "Java Download", and clicking that takes you to **another** page where there's a "Free Java Download" button (at least for me on OSX). Nitpicky, I know, but I wouldn't want anybody getting confused by not seeing a "free Java download" button when they may be expecting it. :) 

So my suggestion would be:

> [...] If you don't have Java installed on your computer, you can download an installer by going to http://java.com in your web browser, and clicking the "Java Download" button; that should take you to a new page with information on the recommended version of Java for your machine, which you can download by clicking the "Free Java Download" button. [..]

**Comment by zkamvar** [comment]
A couple of notes from testing this on an older windows laptop (Windows 10 version 1909):

I was able to download Java and OpenRefine with little issue. I was also able to run OpenRefine and load the data set from this lesson. 

One thing that may be confusing is if the learner follows a link on the java page that says "Do I have Java?", and clicks on the "Verify Java" button. It's a known issue that browsers like Chrome and Firefox have disabled Java plugin support, so this button will give a message that says "We are unable to verify if Java is currently installed and enabled in your browser". This is what I got even though I was able to run OpenRefine with no problem. Moreover, when I uninstalled Java and tried to run OpenRefine, it opened the page to download Java in my browser instead of launching. 

It may be worthwhile to instruct the learners to test their installation by running OpenRefine. If the website for downloading java opens, then they should try to install java or troubleshoot their installation first.
